packages/cqrs/analysis_options.yaml

```diff
@@ -1,5 +1,6 @@
-include: package:lint/analysis_options_package.yaml
+include: package:leancode_lint/analysis_options.yaml
```

Comment:
You might want to use the `_package` variant

Reply:
Why should I do this?
Also, after appending `_package` to this line, I'm getting warnings for lints that are not included in `leancode_lint`.
<img width="999" alt="Screenshot 2021-11-28 at 2 46 59 PM" src="https://user-images.githubusercontent.com/40357511/143770546-6f72e2e0-4d35-4c39-abf4-d3baba271ea4.png">

Reply:
They are: https://github.com/leancodepl/flutter_corelibrary/blob/b00924d127ce80b1aefba703cd3f16d3919cfea7/packages/leancode_lint/lib/analysis_options_package.yaml#L6

the `_package` variant is supposed to be used for packages, cc @RobertOdrowazLNCD that was the point right?

Reply:
Yeah I found out after a while that there's also the `_package`. Makes perfect sense, thanks.